ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -293,6 +308,8 @@ def _fetch_tests_to_skip(self):
             return
 
         self._test_suites_to_skip = []
+        if TEST_SKIPPING_LEVEL == SUITE:
```

Comment:
Why is this necessary, given that the preceding line does the same thing unconditionally?

Reply:
Indeed, sorry, bad conflict resolution